lua/ai/init.lua

```diff
@@ -241,7 +203,9 @@ function M.setup(opts)
       M.opts[k] = v
     end
   end
-  assert(M.opts.api_key ~= nil and M.opts.api_key ~= '', 'api_key is required')
+
+  assert(M.opts.gemini_api_key ~= nil and M.opts.gemini_api_key ~= '', 'gemini_api_key is required')
+  assert(M.opts.chatgpt_api_key ~= nil and M.opts.chatgpt_api_key ~= '', 'chatgpt_api_key is required')
```

Comment:
Can we make it optional?
Here is my idea:
- add a `providers` option for each prompt
- check the keys when sending the prompt
- show an error in the panel if no available key is found

```lua
local default_prompts = {
  prompt1: {
    providers: ['gemini', 'openai'],
  },
  prompt2: {
    providers: ['gemini'],
  },
}
```